perf/benches/dedup.rs

```diff
@@ -22,20 +21,14 @@ fn test_packet_with_size(size: usize, rng: &mut ThreadRng) -> Vec<u8> {
 
 fn do_bench_dedup_packets(bencher: &mut Bencher, mut batches: Vec<PacketBatch>) {
     // verify packets
-    let mut bloom: AtomicBloom<&[u8]> = Bloom::random(1_000_000, 0.0001, 8 << 22).into();
+    let mut deduper = sigverify::Deduper::new(1_000_000, 2_000);
     bencher.iter(|| {
-        // bench
-        sigverify::dedup_packets(&bloom, &mut batches);
-
-        // reset
-        bloom.clear_for_tests();
-        batches.iter_mut().for_each(|batch| {
-            batch
-                .packets
-                .iter_mut()
-                .for_each(|p| p.meta.set_discard(false))
-        });
-    })
+        let _ans = deduper.dedup_packets(&mut batches);
+        deduper.reset();
```

Comment:
This only resets if it's saturated or has reached the max age right? I think you'll need to force reset to get proper benchmarks

Reply:
reset is amortized every 500k unique packets or 2s.  so (500k * 98ns + reset time) / 500k = 98.6 ns